 class d13trycatchexp3
{
public static void main(String args[])
{
try
{
int data=50/0;
}
System.out.println("rest of the code...");
{
System.out.println(e);
}
catch(ArithmeticException e)
}
}